phpBB/phpbb/db/doctrine/case_insensitive_string.php

```diff
@@ -15,6 +15,8 @@
 
 use Doctrine\DBAL\Platforms\AbstractPlatform;
 use Doctrine\DBAL\Types\Type;
+use phpbb\db\middleware\postgresql\phpbb_postgresql_platform;
```

Comment:
I'd rather opt for this instead of adding `phpbb`:

```suggestion
use phpbb\db\middleware\postgresql\platform as postgresql_platform;
```

Reply:
Same applies to the `phpbb_*_driver` and `phpbb_*_middleware` classes?